eslint-config-ibmdotcom/index.js

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
@annawen1 @kennylam just a thought, maybe not part of this PR. Any thought on publishing our eslint config to npm?

Reply:
yeah we can do that what about stylelint as well?

Reply:
possibly...it would bring consistency for sure, we can also have these as available packages to adopters too if they so choose, though what we are providing is very opinionated

Reply:
I thought about this. It can't hurt and would save us from duping it in future projects like this. I'm all for it.